notebooks/compendium/teleportation/index.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>I think the two paragraphs on laser pointers and globes are a bit convoluted and could probably be summed up by explaining that location requires two coordinates (latitude and longitude). Most readers should be familiar with this already.</p><p> </p><p>(Warning: I'm not 100% sure on the next bit, so take with a pinch of salt) I think we should be careful about the analogy of breaking something down and re-creating it. There's a key difference between teleportation, and re-producing the qubit state from scratch.</p><blockquote>Alice takes her information and reduces it down to two pieces of information. Then Bob uses that information to recreate the information for himself.</blockquote><p>Especially with the spherical coordinates example beforehand, this explanation to me sounds a bit like Alice explains to Bob how to transform $|0\rangle \rightarrow |\psi\rangle$. Or like we could use Alice's measurements to reconstruct $|\psi\rangle$ on any qubit. Ultimately the important difference is the shared state.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/textbook/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>